public/scss/pages/Article.module.scss

```
@@ -52,6 +52,16 @@
       color: $color_black_text;
       margin: 0;
     }
+
+    img{
+      max-width: 100%;
+      height: auto;
+    }
+
+    iframe,
+    video{
+      max-width: 100%;
+    }
```

boleh dikasih enter juga mba?
```suggestion
    video
    {
      max-width: 100%;
    }
```